lib/gist_storage.sh

```
@@ -0,0 +1,105 @@
+
+# Get gist count
+get_gist_count() {
+    init_gist_metadata
+    jq 'length' "$GIST_METADATA_FILE"
+}
+
+# Format gist for display
+# Usage: format_gist_display <gist_json>
+format_gist_display() {
+    local gist_json="$1"
+    local description=$(echo "$gist_json" | jq -r '.description')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)